static/app/views/starfish/queries/useSpanMetricsSeries.spec.tsx

```diff
@@ -188,4 +188,89 @@ describe('useSpanMetricsSeries', () => {
       )
     );
   });
+
+  it('rolls single-axis responses up into a series', async () => {
```

Comment:
kind of feels like this test case is covered by `queries for current selection`? But I guess this is more specific so maybe it's fine.

Reply:
Ah yeah, that's true! I can remove from the "current selection" test, since that one should only test the URL parameters, rather than the unrolling behaviour